server/container/guice/protocols/jmap/src/main/java/org/apache/james/jmap/rfc8621/RFC8621MethodsModule.java

```diff
@@ -69,4 +80,16 @@ Authenticator provideAuthenticator(MetricFactory metricFactory,
             basicAuthenticationStrategy,
             jwtAuthenticationStrategy);
     }
+
+    @Provides
+    @Singleton
+    JmapRfcConfiguration provideConfiguration(PropertiesProvider propertiesProvider) throws ConfigurationException, MalformedURLException {
+        try {
+            Configuration configuration = propertiesProvider.getConfiguration("jmap");
+            return new JmapRfcConfiguration(new URL(configuration.getString("jmap-rfc-8621-base-path")));
```

Comment:
You put logic in a guice module, and it is bad as it is hard to test it.

I would prefer having a :

```
static JmapRfc8621Configuration from(Configuration conf) { ... }
```

In JmapRfc8621Configuration class.